tests/specs/T476--StartPage-PageChecks.spec.js

```diff
@@ -0,0 +1,35 @@
+const loginTermsPage = require('../pageObject/loginTermsPage.page')
+const startPage = require('../pageObject/startPage.page')
+
+describe('Create or Import first wallet', () => {
+  beforeAll(() => {
+    loginTermsPage.open()
+    loginTermsPage.checkboxes.forEach((checkbox) => {
+        checkbox.$('label').click()
+    loginTermsPage.confirmAndContinueButton.click();
```

Comment:
Weird indentations here too. Maybe we should set up some check-up or autocorrect tool for you?